src/main/java/WayofTime/alchemicalWizardry/common/tweaker/MineTweakerIntegration.java

```diff
@@ -17,8 +22,24 @@ public static void register() {
         MineTweakerAPI.registerClass(FallingTower.class);
         MineTweakerAPI.registerClass(HarvestMoon.class);
 
-        MineTweakerImplementationAPI.onRollbackEvent(new HandleLateAdditionsAndRemovals());
         MineTweakerImplementationAPI.onPostReload(new HandleLateAdditionsAndRemovals());
+        // GTNH-Only features (post 3.1.0)
+        final ArtifactVersion mtVersion = Loader.instance().getIndexedModList().get(MineTweakerMod.MODID)
+                .getProcessedVersion();
+        try {
+            if (VersionRange.createFromVersionSpec("[3.2.4,)").containsVersion(mtVersion)) {
+                GTNHIntegration.register();
+            }
+        } catch (InvalidVersionSpecificationException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static class GTNHIntegration {
+
+        public static void register() {
+            MineTweakerImplementationAPI.onRollbackEvent(new HandleLateAdditionsAndRemovals());
```

Comment:
Should probably add a note/log that late additions and removals probably won't work correctly without the NH CT, or at the very least not the undo/rollback.